core/controllers/profile.py

```diff
@@ -89,8 +89,15 @@ def get(self, username):
             edited_exp_summary_dicts = (
                 summary_services.get_displayable_exp_summary_dicts_matching_ids(
                     user_contributions.edited_exploration_ids))
+            ### function to determine whether current user is same as user on page ###
+            if (self.username == username):
+                profile_is_of_current_user = True
+            if (self.username != username):
+                profile_is_of_current_user = False
+
```

Comment:
Actually for those lines above, it might be more efficient to write it as `profile_is_of_current_user = (self.username == username)`; however, @seanlip can correct me on this.